node/src/accountant/big_int_processing/test_utils.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+pub(in crate::accountant::big_int_processing) mod mod_restricted {
```

Comment:
Find a term or phrase which could be used repeatedly in this position. ("mod_restricted")